packages/fast-tooling-react/src/data-utilities/generate.ts

```diff
@@ -0,0 +1,137 @@
+import { get } from "lodash-es";
+import { CombiningKeyword, DataType, PropertyKeyword } from "./types";
+import { ChildOptionItem } from ".";
+import { getChildOptionBySchemaId } from "./location";
+
+/**
+ * This file contains all functionality for generating data
+ */
+
+const exampleString: string = "example text";
+
+/**
+ * Gets a single example from a schema
+ */
+function getDataFromSchema(schema: any, childOptions?: ChildOptionItem[]): any {
+    if (isObjectDataType(schema)) {
+        const exampleData: any = {};
+
+        if (hasRequired(schema)) {
+            for (const requiredItem of schema.required) {
+                const propertyType: PropertyKeyword = get(
+                    schema,
+                    `${PropertyKeyword.properties}.${requiredItem}`
+                )
+                    ? PropertyKeyword.properties
+                    : PropertyKeyword.reactProperties;
+
+                switch (propertyType) {
+                    case PropertyKeyword.properties:
+                        exampleData[requiredItem] = getDataFromSchema(
+                            schema[propertyType][requiredItem]
+                        );
+                        break;
+                    case PropertyKeyword.reactProperties:
+                        exampleData[requiredItem] = getReactChildrenFromSchema(
+                            schema[propertyType][requiredItem],
+                            childOptions
+                        );
```

Comment:
did you not add a `break;` here intentionally?

Reply:
Nope, thanks for catching that!